packages/terra-brand-footer/src/BrandFooter.module.scss

```diff
@@ -80,27 +80,27 @@
 
     /* stylelint-disable selector-max-compound-selectors */
     // A '/' isn't needed to separate an item from the header it follows
-    .list-header + .menu::before {
+    .list-header + .menu::before { 
```

Comment:
nit: unnecessary whitespace

Reply:
Resolved in this commit: https://github.com/cerner/terra-framework/pull/439/commits/3593894bab994b5d7bb4e09db4d380b0f8c177a6